app/views/users/_info.haml

```diff
@@ -107,4 +107,4 @@
       .medium-4.large-2.columns
         %label Privileges:
       .medium-8.large-10.columns
-        %p= user.list_current_privileges
+        %p= user.current_membership&.decorate&.list_privileges
```

Comment:
It's been [almost a year now since this bug was reported to Hound](https://github.com/houndci/hound/issues/1121). Ugh.